edge-util/src/Microsoft.Azure.Devices.Edge.Storage/UpdatableEncryptedStore.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Storage
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Newtonsoft.Json;
+
+    public class UpdatableEncryptedStore<TK, TV> : EncryptedStore<TK, TV>
```

Comment:
What does "updateable" mean here?

Reply:
It means that it allows updating an un-encrypted store to an encrypted store. This is to allow upgrading existing Edge deployments, where the Twin store is not encrypted to using an Encrypted twin store. 
I should add a comment explaining the same.